DynamicSmoothedGUI.py

```diff
@@ -392,6 +448,7 @@ def updateGridSmoothed(self):
                     self.visitedSet.add(self.curr_tile)
 
                     self.visibilityDraw(lidar_data)
+                    self.drawC1C0()
```

Comment:
remove

Reply:
tried removing this, but it made the path draw on top of c1c0 which was not good, so i don't think this should be removed

Reply:
Got it then maybe remove the Draw C1C0 on line 342? I feel like it is drawing C1C0 twice. It makes sense tho that you have to draw C1C0 after you draw the path, so you can draw C1C0 here and then remove the first draw that is drawn at every iteration on line 342. 

If I am wrong then my b, whatever you know works is good!

Reply:
Makes sense then try and remove first call